### PR TITLE
network08/Fix_wording

### PR DESCRIPTION
**Pull request by denstorti**
# Description

network08/Fix_wording

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [x] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network08/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Your ACL rule inbound and outbound should look like this:
 
 - What is the difference between inbound x outbound?
 - What is going to happen by default? Everything allowed or denied?
-- How would you block SSH connection comming from the network `172.16.200.0/24`?
+- How would you block SSH an incoming connection from a network `172.16.200.0/24`?
```
Comment (kikobr82) [inline classes/02class/exercises/c02-network08/README.md:19]:
```suggestion
- How would you block an incoming SSH connection from a network `172.16.200.0/24`?
```

**Comment by denstorti** [comment]
Done

File: classes/02class/exercises/c02-network08/ANSWER.md [hunk 2]
```diff
@@ -30,4 +30,4 @@ Add your commands and their outputs here
 
 <!-- Don't change anything below this point-->
 ***
-Answer for exercise [c02-network08](https://github.com/devopsacademyau/academy/blob/893381c6f0b69434d9e8597d3d4b1c17f9bc1371/classes/02class/exercises/c02-network08/README.md)
\ No newline at end of file
+Answer for exercise [c02-network08](https://github.com/devopsacademyau/academy/blob/80a940b39bc3ae40378abe7af015cb3c207463f6/classes/02class/exercises/c02-network08/README.md)
```
Comment (christina-roperto) [inline classes/02class/exercises/c02-network08/ANSWER.md:33]: sorry to meddle, but maybe the link below is more precise? https://github.com/devopsacademyau/academy/blob/80a940b39bc3ae40378abe7af015cb3c207463f6/classes/02class/exercises/c02-network08/ANSWER.md

File: classes/02class/exercises/c02-network08/README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Everything is to be done preferable on Sydney region `ap-southeast-2`
 
 #### Look for the network ACL created for your VPC and answer the following:
 
-Your ACL rule inbound and outbound should look like this:
+Your ACL has both inbound and outbound rules as follow:
 
 |Rule#|Type|Protocol|Port Range|Source|Allow/Deny|
 |-|-|-|-|-|-|
```
Comment (christina-roperto) [inline classes/02class/exercises/c02-network08/README.md:12]: again, sorry for the interruption, for tables it should use 3 or more hyphens eg: |---|---|---|---|---|---| 
reference: https://www.markdownguide.org/extended-syntax

File: classes/02class/exercises/c02-network08/ANSWER.md (hunk 2 above)
Comment (roperto) [reply classes/02class/exercises/c02-network08/ANSWER.md:33]: I think the link is for the questions of the exercise, not the answers (the files that is being edited).

File: classes/02class/exercises/c02-network08/README.md (hunk 3 above)
Comment (roperto) [reply classes/02class/exercises/c02-network08/README.md:12]: GitHub "understands" one hyphen only, but other renderers (like my IDE which is set to strict mode) does not.

**Pull request merged**